src/config.rs

```diff
@@ -17,18 +19,29 @@ where
     Self: Sized + Serialize + DeserializeOwned + Default,
 {
     fn load() -> Result<Self, ConfyError> {
-        confy::load::<Self>(*PROJECT_NAME, Self::config_name())
+        confy::load_path::<Self>(Self::get_config_path())
     }
 
     fn store(&self) -> Result<(), ConfyError> {
-        confy::store(*PROJECT_NAME, Self::config_name(), self)
+        confy::store_path(Self::get_config_path(), self)
     }
 
     fn clear() -> Result<(), ConfyError> {
-        confy::store(*PROJECT_NAME, Self::config_name(), Self::default())
+        confy::store_path(Self::get_config_path(), Self::default())
     }
 
     fn config_name() -> &'static str;
+
+    fn get_config_path() -> PathBuf {
+        if !CONFIG_PATH.is_empty() {
+            Path::new(&(*CONFIG_PATH)).join(format!("{}.yml", Self::config_name())).to_path_buf()
+        } else {
+            match confy::get_configuration_file_path(*PROJECT_NAME, Self::config_name()) {
+                Ok(p) => PathBuf::from(p.to_str().unwrap_or_default()).to_owned(),
+                Err(_e) => PathBuf::from("").to_owned()
```

Comment:
Shouldn't we panic if there is no known config path at this point?

Reply:
Yeah, probably right

Reply:
Added panic